src/sql/inspect.py

```diff
@@ -231,13 +232,18 @@ class TableDescription(DatabaseInspection):
     """
 
     def __init__(self, table_name, schema=None) -> None:
-        util.is_table_exists(table_name, schema)
+        with_ = util._check_table_exists(table_name, schema)
 
         if schema:
             table_name = f"{schema}.{table_name}"
 
+        if with_:
+            with_clause = str(store.render("", with_=with_))
```

Comment:
What if we call `store.render` using `query` = `SELECT * FROM {table_name} WHERE 1=0"`, and `with_` = `with_`?
Does it throw any error?

Reply:
I replaced 
`         columns_query_result = sql.run.raw_run(
            Connection.current, f"{with_clause} SELECT * FROM {table_name} WHERE 1=0"
         )
`
with 
`
        columns_query_result = sql.run.raw_run(
        Connection.current, str(store.render(f"SELECT * FROM {table_name} WHERE 1=0", with_=with_))
        ) 
 `
It does not throw an error.

Reply:
Can we do this for all the queries instead of prepending `with_clause`.

Reply:
Made this change, now we do not prepend `with_clause` anywhere.